Sources/NIOCore/BSDSocketAPI.swift

```diff
@@ -66,6 +66,15 @@ private let sysInet_ntop: @convention(c) (CInt, UnsafeRawPointer?, UnsafeMutable
 private let sysInet_pton: @convention(c) (CInt, UnsafePointer<CChar>?, UnsafeMutableRawPointer?) -> CInt = inet_pton
 #endif
 
+// Work around SO_TIMESTAMP/SO_RCVTIMEO being awkwardly defined in glibc.
+#if os(Android) && arch(arm)
+let SO_RCVTIMEO = SO_RCVTIMEO_OLD
+let SO_TIMESTAMP = SO_TIMESTAMP_OLD
+#elseif os(Linux)
+let SO_TIMESTAMP = CNIOLinux_SO_TIMESTAMP
```

Comment:
CI is very unhappy -- it looks like we're missing the CNIOLinux import?

Reply:
Oh yup, seems likely.